Modules/Core/Transform/include/itkTransformGeometryImageFilter.hxx

```diff
@@ -73,21 +74,12 @@ TransformGeometryImageFilter<TInputImage, TOutputImage>::GenerateData()
   {
     newOriginPoint[i] = newOriginVector[i];
   }
-  m_OutputImage->SetOrigin(newOriginPoint);
-  m_OutputImage->SetDirection(inverseRotation * this->GetInput()->GetDirection()); // NewDC = [R^-1][DC]
+  outputPtr->SetOrigin(newOriginPoint);
+  outputPtr->SetDirection(inverseRotation * this->GetInput()->GetDirection()); // NewDC = [R^-1][DC]
 
-  this->GraftOutput(m_OutputImage);
+  this->GraftOutput(outputPtr);
 }
 
-template <typename TInputImage, typename TOutputImage>
-void
-TransformGeometryImageFilter<TInputImage, TOutputImage>::PrintSelf(std::ostream & os, Indent indent) const
```

Comment:
Why remove this?

Reply:
There are no longer an ivars to print :)